src/Inkluzitron/Services/UsersService.cs

```diff
@@ -22,23 +22,33 @@ public UsersService(DatabaseFactory databaseFactory)
             DbContext = DatabaseFactory.Create();
         }
 
-        public async Task<User> GetUserDbEntityAsync(IUser user)
+        public Task<User> GetUserDbEntityAsync(IUser user)
```

Comment:
Zbavil bych se toho vlastního kontextu úplně, to jen čeká než si na tom jinde vylámeme zuby znovu.